### ISSUE TITLE
Gtk4 Vala examples from @aeldemery

### ISSUE THREAD

**Issue by Prince781** [nahuelwexd/vala-website#7]
@aeldemery has been working on a wonderful set of examples for Gtk4 with Vala. Perhaps we could find a way to include some of them.

**Comment by colinkiama** [comment]
@lw64

**Comment by colinkiama** [comment]
The docs site now linked to this one seems to be a great place to include that.

Where can we find those examples?

**Comment by Prince781** [comment]
@colinkiama you can find them on @aeldemery's profile

**Comment by lw64** [comment]
They are looking really good. But I think it would be cool to make the structure and coding style of them more vala-like and then make tutorials out of it.

**Comment by colinkiama** [comment]
I agree. These would be really great as tutorials in the docs website

**Comment by colinkiama** [comment]
These could be a series of blog posts.

@aeldemery would you be okay with that?

**Comment by aeldemery** [comment]
Hallo all,
Thank you for your efforts you put for the community. 
You can use the code as you like.
Unfortunately I can't entertain my coding "hobby" anymore as my job got more demanding and my family became bigger.
Hopefully I could resume it one day.
Thanks again.

**Comment by colinkiama** [comment]
Thank you
